requirements-dev.txt

```diff
@@ -4,355 +4,125 @@
+coveralls==1.8.2          # via -r requirements-dev.in
+cssselect==1.1.0          # via -r requirements.in
+diff-match-patch==20200713  # via -r requirements.in
+docopt==0.6.2             # via coveralls
+docutils==0.16            # via codechat, recommonmark, runestone, sphinx
+easyprocess==0.3          # via pyvirtualdisplay
+entrypoints==0.3          # via altair
+flake8==3.8.4             # via -r requirements-dev.in
+flask-basicauth==0.2.0    # via locust
+flask==1.1.2              # via flask-basicauth, locust
+future==0.18.2            # via -r requirements.in
+gevent==21.1.2            # via celery, geventhttpclient, locust
+geventhttpclient==1.4.4   # via locust
+greenlet==1.0.0           # via gevent
+html5validator==0.3.3     # via -r requirements-dev.in
+httplib2==0.18.1          # via oauth2
+idna==2.10                # via requests, runestone
+imagesize==1.2.0          # via runestone, sphinx
+iniconfig==1.1.1          # via pytest
+isort==5.7.0              # via pylint
+itsdangerous==1.1.0       # via flask
+jinja2==2.11.3            # via altair, flask, runestone, sphinx
+jsonschema==3.2.0         # via altair
+kombu==5.0.2              # via celery
+lazy-object-proxy==1.4.3  # via astroid
+locust==1.4.3             # via -r requirements-dev.in
+lxml==4.6.2               # via -r requirements.in, codechat, runestone
+markupsafe==1.1.1         # via jinja2, runestone
+mccabe==0.6.1             # via flake8, pylint
+mock==4.0.3               # via -r requirements-dev.in
+msgpack==1.0.2            # via locust
+mypy-extensions==0.4.3    # via black
+numpy==1.20.0             # via -r requirements.in, altair, pandas
+oauth2==1.9.0.post1       # via -r requirements.in
+packaging==20.9           # via bleach, pytest, runestone, sphinx
+pandas==1.2.1             # via -r requirements.in, altair
+pathlib2==2.3.5           # via -r requirements.in
+pathspec==0.8.1           # via black
+paver==1.3.4              # via -r requirements.in, runestone, sphinxcontrib-paverutils
+pip-tools==5.5.0          # via -r requirements-dev.in
+pluggy==0.13.1            # via pytest
+polling2==0.4.6           # via -r requirements-dev.in
+prompt-toolkit==3.0.14    # via click-repl
+psutil==5.8.0             # via locust
+psycopg2-binary==2.8.6    # via -r requirements.in
+py==1.10.0                # via pytest
+pycodestyle==2.6.0        # via flake8
+pycparser==2.20           # via cffi
+pyflakes==2.2.0           # via flake8
+pygments==2.7.4           # via codechat, runestone, sphinx
+pylint==2.6.0             # via -r requirements.in
+pyparsing==2.4.7          # via packaging, runestone
+pyrsistent==0.17.3        # via jsonschema
+pytest==6.2.2             # via -r requirements-dev.in
+python-dateutil==2.8.1    # via -r requirements.in, pandas
+pytz==2021.1              # via -r requirements.in, babel, celery, pandas, runestone
+pyvirtualdisplay==2.0     # via -r requirements-dev.in
+pywin32==300              # via locust
```

Comment:
pywin32 does not install on mac or windows....

Reply:
Or maybe we should not `pip-compile` on windows ??

Reply:
Grrr. It looks like locust brings this in unconditionally on Windows. I added an environment marker to try and work around it. Hopefully, that will work.